ansible.cfg

```diff
@@ -0,0 +1,3 @@
+[defaults]
+
+library = /home/vagrant/isic_archive/ansible/library
```

Comment:
Eventually, I'll get Conda support pushed upstream to Ansible, and this won't be necessary anymore. Good workaround for now.